plugins/PrepareStage/PrepareMenu.qml

```diff
@@ -13,6 +13,9 @@ Item
     signal showTooltip(Item item, point location, string text)
     signal hideTooltip()
 
+    property bool isNetworkPrinter: Cura.MachineManager.activeMachineNetworkKey != ""
+    property bool printerConnected: Cura.MachineManager.printerConnected
```

Comment:
Better use aliases for these.

Reply:
I better moved the checks to the `QuickConfigurationSelector` in c8bdf7321c4a888025001064f11c852097404264